wp-content/civi-extensions/goonjcustom/Civi/CollectionBaseService.php

```diff
@@ -177,11 +177,6 @@ public static function collectionBaseTabset($tabsetName, &$tabs, $context) {
         'module' => 'afsearchEventVolunteer',
         'directive' => 'afsearch-event-volunteer',
       ],
-      'materialContribution' => [
-        'title' => ts('Material Contribution'),
-        'module' => 'afsearchCollectionCampMaterialContributions',
-        'directive' => 'afsearch-collection-camp-material-contributions',
-      ],
```

Comment:
Since the material contribution form may change in the future for dropping center, it's beneficial to keep the forms separate for the collection camp and the dropping center. Therefore, I have removed the combined version, and both will now be added individually for the collection camp and the dropping center. Please let me know if you have any further questions.